sql/core/src/main/scala/org/apache/spark/sql/SparkSession.scala

```diff
@@ -323,7 +323,7 @@ class SparkSession private(
    *  // |-- age: integer (nullable = true)
    *
    *  dataFrame.createOrReplaceTempView("people")
-   *  sparkSession.sql("select name from people").collect.foreach(println)
+   *  sparkSession.sql("select name from people").show
```

Comment:
I agree. Just for the record, `collect.foreach(println)` should not be "endorsed" in Spark 2's official docs as too RDD-ish.